src/lib/accessibility/Behaviors/Popup/popupBehavior.ts

```diff
@@ -11,6 +11,7 @@ import * as _ from 'lodash'
  * Adds attribute 'aria-disabled=true' to 'trigger' component's part if 'disabled' property is true. Does not set the attribute otherwise.
  */
 const popupBehavior: Accessibility = (props: any) => {
+  const normalizedOn = _.isArray(props.on) ? props.on : [props.on]
```

Comment:
would suggest name like `onAsArray` - that would be much more expressive than general 'normalize'